src/MPU6050_light.cpp

```diff
@@ -50,32 +44,48 @@ byte MPU6050::writeData(byte reg, byte data){
 
 // This method is not used internaly, maybe by user...
 byte MPU6050::readData(byte reg) {
-  wire->beginTransmission(MPU6050_ADDR);
+  wire->beginTransmission(readReg);
   wire->write(reg);
   wire->endTransmission(true);
-  wire->requestFrom(MPU6050_ADDR, 1);
+  wire->requestFrom(readReg, 1);
   byte data =  wire->read();
   return data;
 }
 
 /* SETTER */
 
+byte MPU6050::switchReadingAddress(){
+	byte status;
+	switch(readReg){
+	case 0x68:
+	  readReg = 0x69;
+	  break;
+	case 0x69:
+	  readReg = 0x68;
+	  break;
+	default: // error
+	  status = 1;
+	  break;
+	}
+	return status;
+}
```

Comment:
This can be simplified since there are only 2 addresses and the attribute is private. I think there is no need to overcheck here.
```cpp
void MPU6050::switchAddress(){
  if(address==0x68){
    address=0x69;
  }else{
    address=0x68;
  }
}
```